moveit_ros/planning/kinematics_plugin_loader/src/kinematics_parameters.yaml

```diff
@@ -4,23 +4,30 @@ kinematics:
     default_value: kdl_kinematics_plugin/KDLKinematicsPlugin,
     description: "Kinematics solver",
     validation: {
-      not_empty<>: []
+      not_empty<>: [],
+      one_of<>: [[kdl_kinematics_plugin/KDLKinematicsPlugin,
+                  prbt_manipulator/IKFastKinematicsPlugin,
+                  bio_ik/BioIKKinematicsPlugin,
+                  lma_kinematics_plugin/LMAKinematicsPlugin,
+                  srv_kinematics_plugin/SrvKinematicsPlugin,
+                  cached_ik_kinematics_plugin/CachedKDLKinematicsPlugin,
+                  cached_ik_kinematics_plugin/CachedSrvKinematicsPlugin ]]
```

Comment:
This is a hard-ish problem. We want this to be an extension point of moveit but we also want to validate that the value set here is a valid kinematics_base plugin. I would probably leave this as just `not_empty` and I will work on writing a validator for you that uses pluginlib to validate that the string your receive here is a valid plugin and not just part of some hard coded set but instead plugins that your system enviroment has.